preprocessors/multistage-textbook-diagram.schema.json

```diff
@@ -0,0 +1,149 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/mutistage-textbook-diagram.schema.json",
+    "type": "object",
+    "title": "Multistage Textbook Diagram Data",
+    "description": "JSON containing pixel coordinates of different stages in a mutistage textbook diagram and the links between them",
+    "definitions": {
+        "normCoord": {
+            "description": "A pair of normalized coordinates.",
+            "type": "array",
+            "items": {
+                "type": "number",
+                "minimum": 0,
+                "maximum": 1
+            },
+            "minItems": 2,
+            "maxItems": 2
+        },
+        "contour": {
+            "type": "object",
+            "description": "A single contour of a segment as detected. Each contour is a separate region.",
```

Comment:
What does this mean?

Reply:
Copied over from semseg schema. Does "A closed region within a segment" make more sense?

Reply:
Not really. You're presenting this as a representation of diagram data, so it's a bit strange to not put this in schema-specific terms.